.github/workflows/progress-tracker.yml

```diff
@@ -75,14 +75,14 @@ jobs:
               if (!nums.length) {
                 return [0, 0, 0, 0, 0]
               }
-              nums.sort((a, b) => a - b)
+              nums.sort()
```

Comment:
Please revert this change, because by default sorting is not numeric. If the number goes in two digits, you will see messy results like this:

```
[3, 11, 222].sort()
> [11, 222, 3]

Reply:
Oh good catch. That's unexpected.